src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/ProjectSystems/CpsPackageReferenceProject.cs

```diff
@@ -133,6 +133,7 @@ private Task<string> GetAssetsFilePathAsync(bool shouldThrow)
         private PackageSpec GetPackageSpec()
         {
             DependencyGraphSpec projectRestoreInfo;
+            //TODO NK - We need the Project System team to plumb through extra details
```

Comment:
Use your active GH issue (or file a new one) to track this and other TODOs. Thus it'd be easy to identify them across the code.